.github/workflows/nextjs.yml

```
@@ -0,0 +1,37 @@
+name: Next.js CI
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  next:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

You can use `actions/checkout@v3` for the latest version of this action.

```suggestion
        uses: actions/checkout@v3
```